### PR TITLE
Add a caution comment for installing OpenCV in Raspberry Pi

### PR DESCRIPTION
**Pull request by YunjinJo**
Issue #2010 
Add a caution comment in the code block to prevent a user from misconceiving.

### CODE REVIEW

File: intermediate_source/realtime_rpi.rst [hunk 1]
```diff
@@ -68,6 +68,7 @@ PyTorch and all the other libraries we need have ARM 64-bit/aarch64 variants so
 .. code:: shell
 
     $ pip install torch torchvision torchaudio
+    # Installing OpenCV will take about 2 hours and nothing shows in the terminal until it finishes.
```
Comment (svekars) [inline intermediate_source/realtime_rpi.rst:71]: @d4l3k is this expected?

Comment (d4l3k) [reply intermediate_source/realtime_rpi.rst:71]: That seems way longer than expected. @YunjinJo  what RPi are you using? Are you using 64bit arm or 32bit? And it's the install that takes a long time not the download?

**Review by d4l3k — APPROVED** [review]

File: intermediate_source/realtime_rpi.rst (hunk 1 above)
Comment (d4l3k) [reply intermediate_source/realtime_rpi.rst:71]: Though, if it builds from the source distribution that might be reasonable... 

Seems like other folks are seeing the same thing: https://raspberrypi-guide.github.io/programming/install-opencv

There's some prebuild rpi wheels at https://www.piwheels.org/project/opencv-contrib-python/ but only for 32bit

**Comment by d4l3k** [comment]
Here's the issues for adding prebuilt aarch64 wheels to opencv python 

https://github.com/opencv/opencv-python/issues/473
https://github.com/opencv/opencv-python/issues/562

**Comment by d4l3k** [comment]
Looks like opencv-python has a aarch64 prebuilt library -- wonder if we should switch to that instead of contrib https://pypi.org/project/opencv-python/#files

**Comment by d4l3k** [comment]
Since we're only using `cv2.VideoCapture` and that's available in opencv-python does seem like we should just switch to that (or at least recommend it)

@YunjinJo have you tried just using `opencv-python`?

**Comment by YunjinJo** [comment]
> Since we're only using `cv2.VideoCapture` and that's available in opencv-python does seem like we should just switch to that (or at least recommend it)
> 
> @YunjinJo have you tried just using `opencv-python`?

@d4l3k  
I tested on my RPI (RPI 4 model B 8GB, RPI OS 64-bit) using `opencv-python` and it works well. I think it would be better to use `opencv-python` than `opencv-contrib-python`.

**Review by svekars — APPROVED** [review]

**Pull request merged**